### ISSUE TITLE
error to insert new item into beta.blogger.com

### ISSUE THREAD

**Issue by GoogleCodeExporter** [barsu8184/google-gdata#26]
```
insert new item into beta.blogger.com,

new item entity is expected, but got a HTTP 302 found status code.

I am using version 1.0.7, and the OS is windows XP SP2

exception throw from request.cs Execute method,
the old www.blogger.com works fine, only beta.glogger.com has trouble.

```

Original issue reported on code.google.com by `typedro...@gmail.com` on 6 Nov 2006 at 3:50

**Comment by GoogleCodeExporter** [comment]
```
there is a problem with the .NET handling of this, it relies on a Gdata session
cookie instead of using the URI that is returned. Need to fix that
```

Original comment by `fman...@gmail.com` on 6 Nov 2006 at 7:18
- Changed state: **Accepted**
- Added labels: **Client-CSharp**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in releaase 1.08, currently in Subversion.
```

Original comment by `fman...@gmail.com` on 10 Nov 2006 at 3:11
- Changed state: **Fixed**
